roles/apache/defaults/main.yml

```diff
@@ -1,5 +1,5 @@
 ---
-os_log_path:
+apache_log_path:
```

Comment:
yes that did come to my mind, but given this was closely tied to apache thought might be good left here. Example: Keystone if has dependency on apache it has to add this role as dependency to use the handlers etc.